deis-cluster-coreos/azuredeploy.json

```diff
@@ -359,8 +366,19 @@
             "publisher": "[variables('imagePublisher')]",
             "offer": "[variables('imageOffer')]",
             "sku": "[variables('imageSKU')]",
-            "version": "latest"
+            "version": "647.2.0"
```

Comment:
I have to get approval for this on our end. We've been working with Jim Zimmerman at MSFT to get this template updated. /cc @jimzim